cfe_internal/update/update_policy.cf

```diff
@@ -80,6 +80,47 @@ bundle agent cfe_internal_update_policy
 
 }
 
+bundle agent cfe_internal_remove_broken_symlink(symlink_path)
+# @brief Checks if the symlink_path symlink points to an existing target
+# @warning Only works for symlinks using absolute paths!
+{
+  vars:
+      "link_target" string => filestat("$(symlink_path)", "linktarget");
+
+  files:
+      "$(symlink_path)"
+        delete => u_tidy,
+        if => not(fileexists("$(link_target)"));
+}
+
+bundle agent cfe_internal_setup_python_symlink(symlink_path)
+# @brief Create the /var/cfengine/bin/python symlink pointing to some installed python (if any)
+{
+  vars:
+      "path" string => getenv("PATH", 1024);
+      "path_folders" slist => splitstring("$(path)", ":", 128);
+
+      "exact_version_globs" slist => maplist("$(this)/python[23]", @(path_folders));
```

Comment:
Would be nice to have a few hash comments throuhgout this section where your finding and ending up with a list of valid pythons to explain what's happening.

Reply:
will do